prow/Makefile

```diff
@@ -72,7 +74,6 @@ bazel-release-push:
 	@echo See https://bazel.build/ for install options.
 	@echo Be sure to setup authentication: https://github.com/bazelbuild/rules_docker#authentication
 	@echo Also run gcloud auth application-default login
-	export STABLE_PROW_REPO=$(REGISTRY)/$(PROJECT)
```

Comment:
Please just change this to PROW_REPO_OVERRIDE

Reply:
the issue here is... each line in makefile is a separate process, which means this export line won't take affect in the `bazel run` below